src/functionalTest/java/uk/gov/hmcts/reform/sscs/functional/tyanotifications/handlers/AdminAppealWithdrawnNotificationsTest.java

```diff
@@ -63,12 +62,15 @@ public void givenCallbackWithSubscriptions_shouldSendEmailSmsAndLetterNotificati
     }
 
     private boolean fetchLetters(int expectedNumLetters, String subscription) {
+        int fetchCount = 0;
         do {
             if (getNumberOfLetterCorrespondence(subscription) == expectedNumLetters) {
                 return true;
             }
+            fetchCount++;
             delayInSeconds(5);
-        } while (true);
+        } while (fetchCount < 5);
```

Comment:
Should 5 and the 3 above be a const somewhere? Maybe shared? Is there any impact on adding the retry?

Reply:
I was just trying to get the flakiness down - probably could remove the retry as the fetch count should cover that much, looking at the code, there is already a MAX_RETRY_ATTEMPTS set in the values so will just use that instead